### ISSUE TITLE
error when analyzing single color images

### ISSUE THREAD

**Issue by edoelas** [AIM-Harvard/pyradiomics#513]
I have been using this library for analizing stone textures (is the best library I have been able to found for this purpose) and some of them where fake textures generated by computer. Two of the textures I analized where completely smooth, just one gray color. When trying to extract features from this images I got a weird error and at the end the way I solve it was to add 1 to the gray value of the first pixel of the image.
Right now I can't reproduce the error but as soon as posible I will do it and give more information.
Thanks!

**Comment by JoostJM** [comment]
@edoelas have you made any progress investigating this issue?
A completely flat texture (i.e. just one gray value) does produce outlier values/errors as the variance is 0, and would lead to 0-division is certain features (mostly, in these cases an arbitrary value is returned)

**Comment by edoelas** [comment]
I just wanted to test if this library suite my needs (and it is a great library, but my needs are really weird) so I just modified the slightly the value of one pixel.
I commented this because if the user is not aware of this problem it may lead to try to find an error that does not exists in its own code. Or even worse, if it is like my case, where I was analyzing three thousand images and two of them for some reason were complete flat, and, in this situation, maybe to my lack of experience, it was hard for me to find the problem.
